### PR TITLE
Fix availability

### PR DESCRIPTION
**Pull request by chrysanthos**

### CODE REVIEW

File: app/Console/Commands/UpdatePharmacyAvailability.php [hunk 1]
```diff
@@ -15,26 +15,39 @@ class UpdatePharmacyAvailability extends Command
...
+
             $this->info('Parsed a total of ' . $counts['rows'] . ' rows (' . $counts['failed'] . ' failed).');
             $this->info('Pharmacies: ' . $counts['pharmacies']['added'] . ' added / ' . $counts['pharmacies']['updated'] . ' updated / ' . $counts['pharmacies']['alreadyFine'] . ' already fine.');
             $this->info('Availabilities: ' . $counts['availabilities']['added'] . ' added / ' . $counts['availabilities']['updated'] . ' updated / ' . $counts['availabilities']['alreadyFine'] . ' already fine.');
+
             $filesparsed++;
-            $this->info('====================');
+
+            $this->info('');
+            $this->info('========================================');
```
Comment (gregkos) [inline app/Console/Commands/UpdatePharmacyAvailability.php:45]: Why tho

**Comment by gregkos** [comment]
You removed the old CSV files, but didn't add the new ones?

**Comment by gregkos** [comment]
Safe to assume these files are actually unchecked?

**Comment by chrysanthos** [comment]
@gregkos Tested locally.

**Review by gregkos — APPROVED** [review]

**Pull request merged**